tests/records/full-record.json

```diff
@@ -143,12 +137,12 @@
       }
     }],
     "references": [{
-      "reference_string": "Nielsen et al,..",
+      "reference": "Nielsen et al,..",
       "identifier": "101.234",
       "scheme": "doi"
     }]
   },
-  "metaext": {
+  "ext": {
     "dwc:collectionCode": "abc",
```

Comment:
Minor: Can you change to also:

```
{
  "ext": {
    "dwc": {
      "...": "..."
    }
  }
}
```

Reply:
Fixed jsons and removed limitations on jsonschema for now, will fix when working on exts.